docroot/profiles/custom/cgov_site/modules/custom/cgov_article/src/CGovArticleConfigOverride.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\cgov_article;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+
+/**
+ * Configuration Override for CGov Article.
+ */
+class CGovArticleConfigOverride implements ConfigFactoryOverrideInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadOverrides($names) {
```

Comment:
This is an example of a ConfigOverride function which enables the cgov_article split when this module is enabled.